modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -468,6 +459,24 @@ export default {
     },
     handleModalAction (action) {
       console.info('Execute modal action', action);
+    },
+    setMoreMenu () {
+      const { moreMenu } = this.moduleOptions;
+
+      const newPiece = {
+        action: 'new',
+        label: {
+          key: 'apostrophe:newDocType',
+          type: this.$t(this.moduleLabels.singular)
+        }
+      };
+
+      this.moreMenu.menu = [
+        ...this.relationshipField && this.moduleOptions.canEdit
+          ? [ newPiece ] : [],
+        ...this.moreMenu.menu,
+        ...Array.isArray(moreMenu) && moreMenu.length ? moreMenu : []
```

Comment:
I'm not sure if we need to check `moreMenu.length`. If it's 0 then you get the empty array either way.

Reply:
That's right 😅